metrics/metrics.go

```diff
@@ -18,12 +18,20 @@ package metrics
 
 import (
 	"github.com/containerd/containerd/version"
-	goMetrics "github.com/docker/go-metrics"
+	metrics "github.com/docker/go-metrics"
+)
+
+var (
+	// GcCounter metrics for garbage collection
+	GcCounter metrics.LabeledCounter
```

Comment:
GcCounter -> gcCounter?

Reply:
each package can define and register their own metrics. The metrics package should not be used in this way to centralize counters

Reply:
Thanks for review! i thought gc metrics should under `containerd` namespace, so i put it under metrics package. Show i move it to gc/scheduler ?